src/sentry/api/serializers/types.py

```diff
@@ -33,3 +33,146 @@ class OrganizationSerializerResponse(TypedDict):
     features: list[str]
     links: _Links
     hasAuthProvider: bool
+
+
+class ReleaseOptional(TypedDict, total=False):
```

Comment:
Can you move these to their own file? Asking because right now you have `api.serializer.types.project` which sounds very generic and not specific to release. It can be used by someone by mistake instead of project serializer. Same as `version` it's not clear it's a release version being in the high level module.